virt/virt-web-console-overview.adoc

```diff
@@ -544,8 +544,8 @@ The number of CPUs is calculated by using the following formula: `sockets * thre
 |SSH using virtctl
 |Click the copy icon to copy the `virtctl ssh` command to the clipboard.
 
-|SSH over NodePort
-|Selecting *Create a Service to expose your VirtualMachine for SSH access* generates an `ssh -p <port>` command. Click the copy icon to copy the command to the clipboard.
+|SSH service type options
+|Select the option for *SSH using LoadBalancer* or *SSH over NodePort*. Both options generate an `ssh -p <port>` command. Click the copy icon to copy the command to the clipboard.
```

Comment:
`SSH using LoadBalancer` is `SSH over LoadBalancer` on UI.
Others are LGTM

Reply:
@gouyang Thanks for looking! The screenshot I was provided of the UI though shows `SSH using LoadBalancer` (see this [Jira/Doc](https://issues.redhat.com/browse/CNV-23849)). Are you able to confirm the exact wording?

Reply:
@danielclowers 
You can find the exact wording in this [Json file](https://github.com/kubevirt-ui/kubevirt-plugin/blob/7f131a3a392a783793a06f23a332e4e7bdfab508/locales/en/plugin__kubevirt-plugin.json#L904). :-)